sdk/storage/storage-queue/src/generated/src/storageClientContext.ts

```diff
@@ -39,7 +39,7 @@ export class StorageClientContext extends coreHttp.ServiceClient {
 
     super(undefined, options);
 
-    this.version = '2019-02-02';
+    this.version = '2018-03-28';
```

Comment:
This should be changed back with a transform

Reply:
Fixed.